### PR TITLE
change image limits url

### PR DESCRIPTION
**Pull request by simon300000**
我没改版本号

### CODE REVIEW

**Comment by ccloli** [comment]
Anyway, thanks for reading this messy code and sending this PR. Maybe I should start to rewrite this script, but not still working in progress, too busy or too lazy or some how...

**Comment by simon300000** [comment]
I'm sorry I made a mistake also, it should be "e-hentai"

I try read your code, It seems like the script that get limit number of exhentai use e-hentai.org/home.php

but download original which from IP might have different IP cause by proxy seems true, so this setting overall is working but that seems a small problem to me.

line 13551
```
function getImageLimits(host, forced){
	var host = host || location.hostname;
	if (host === 'exhentai.org') {
		host = 'e-hentai.org';
	}
	var url = 'http://' + host + '/home.php';
```

**Comment by ccloli** [comment]
@simon300000 Thanks for your commit, however it's still working in progress and it's a new feature, so please do not push the commit to that PR. Since you have do it, please try the following shell to reset your work. The code has been tested locally, but if you confuse that this will destroy what you are doing, `cp` the git repo folder to backup. 

```bash
git stash                         # 0. <opt> if you have modified your local file
git reset --soft e97a4ff6         # 1. reset to previous commit
git checkout -b patch-audio-test  # 2. create a new branch based on current branch
git reset --hard f657c39f         # 3. reset to the commit you forked
git cherry-pick 3d8f17bb          # 4. cherry-pick your commit that you just added
git checkout master               # 5. return to your master branch
git push --force                  # 6. force pushed resetted branch to reset PR
git checkout patch-audio-test     # 7. return to the branch you are working
git push                          # 8. push the new branch to GitHub and send a new PR
git stash apply                   # 9. <opt> apply your local change you stashed
```

If you have new commits, change `3d8f17bb` to your new commit hash. You can run `git log` to see the hash of your latest commit.

**Before doing step 6, run `gitk --all` (or ` git log --graph --all` if it said you don't have `gitk`) and check if your git tree is the same as below:**

image
_Note: The tree should split from my last commit._

If not, **stop doing the following step**, run `git reset --hard 3d8f17bb` to reset what you are doing, and comment what happens here.

**Comment by simon300000** [comment]
should we use git flow →_→？
Thank

**Comment by ccloli** [comment]
Okay, looks fine now, I'll merge this PR. If you have any news about your audio test, open a new PR. Thanks for your help. 😝 

BTW just fixed a critical bug about File System, changing the code written in nearly 2 years ago (03742f2) is really a tough work... Maybe it's time to refactor (rewrite) to v2. 😕

**Comment by simon300000** [comment]
我觉得需要用一些比较“专业”的node编译软件了，这个脚本功能多了变得比较负载 代码不压缩够大的。。。（虽然没必要压缩）

**Comment by simon300000** [comment]
I don't have much to do with the audio, I made the audio keep playing if save is forced, and stop when thing is done.
I tested in my Safari it working fine, but I don't have the problem so I don't know is that gonna actually fix it or not, so I might need somebody who have the problem to test for me.

File: e-hentai-downloader.user.js [hunk 1]
```diff
@@ -13429,7 +13429,7 @@ function showSettings() {
...
 				<div class="g2"><label><input type="checkbox" data-ehd-setting="replace-with-full-width"> Replace forbidden letters as full-width letters instead of dash (-)</label></div>\
 				<div class="g2"><label><input type="checkbox" data-ehd-setting="force-pause"> Force drop downloading images data when pausing download</label></div>\
-				<div class="g2"><label><input type="checkbox" data-ehd-setting="image-limits-both"> I\'m in China and/or using proxy to visit exhentai.org so my image limits on ExHentai is incorrect</label></div>\
+				<div class="g2"><label><input type="checkbox" data-ehd-setting="image-limits-both"> I\'m in China and/or using proxy to visit e-hentai.org so my image limits on ExHentai is incorrect</label></div>\
```
Comment (ccloli) [inline e-hentai-downloader.user.js]: Looks like this option is useless, all the requests that to `g.e-hentai.org` and `r.e-hentai.org` are redirect to `e-hentai.org`, so this option will be removed in next version.

**Pull request merged**